draft-meynell-panrg-scion-research-questions.md

```diff
@@ -276,6 +276,18 @@ When a client contacts a server, it is usually understood that it wants the serv
 
 There are some nuances: Usually the server's API will store the initial address of the client to be used through all the session. We might need to take this into account.
 
+A related question: how long before expiration should we still use a path? How do we handle that?
+
+Do we actually need to solve this reverse path refresh problem?
+
+* CONTRA: It is probably rare that a server needs to send data for a long time without the client ever answering back.
```

Comment:
Idk, we have use cases like video streaming, maybe?

Reply:
I assume video streaming also has built-in keep-alive to stop sending if the client is disconnected?
I'll change it to "... send data for a long time without the **[application layer protocol requiring the]** client **[to]** ever answer[] back.